tests/component-plugin/src/main/java/org/mule/functional/api/extension/TestComponentExtensionLoadingDelegate.java

```diff
@@ -78,6 +74,8 @@ public void accept(ExtensionDeclarer extensionDeclarer, ExtensionLoadingContext
     declareProcessor(extensionDeclarer);
     declareInvocationCounter(extensionDeclarer);
     declareAssert(extensionDeclarer);
+    declareParameterInterceptor(extensionDeclarer);
+    declareDumpInterceptedParameters(extensionDeclarer);
```

Comment:
is this being used? We should aim to use the test-extension instead of adding more stuff in here.